azure/templates/deploy-service.yml

```diff
@@ -29,6 +22,27 @@ parameters:
     default: []
 
 steps:
+  - bash: |
+      ls -R
+      if [[ -f ecs-proxies-deploy.yml ]]; then
+        echo "##vso[task.setvariable variable=build_containers]true"
+      else
+        echo "##vso[task.setvariable variable=build_containers]false"
+      fi
+    workingDirectory: ${{ parameters.service_name }}/$(SERVICE_ARTIFACT_NAME)
+    displayName: "Check for ECS proxy definitions"
+
+  - bash: |
+      git clone https://github.com/NHSDigital/api-management-utils.git $(Pipeline.Workspace)/s/${{ parameters.service_name }}/$(SERVICE_ARTIFACT_NAME)/utils
+      cd $(Pipeline.Workspace)/s/${{ parameters.service_name }}/$(SERVICE_ARTIFACT_NAME)/utils
+      git checkout apm-1313-ecs-deploy-in-common-pipelines
+    displayName: Clone utils
```

Comment:
Shouldn't we use the resources way of getting the utils artifact. Also, `git checkout apm-1313-ecs-deploy-in-common-pipelines` will need removing.

Reply:
Ah - as part of this ticket we tried to do that, but when checking it out it would clear the working folder, so reverted back to this way. And cool - will remove.